logger.go

```diff
@@ -26,5 +26,3 @@ type defaultLogger struct{}
 func (l *defaultLogger) Print(v ...interface{})                 { log.Print(v...) }
 func (l *defaultLogger) Printf(format string, v ...interface{}) { log.Printf(format, v...) }
 func (l *defaultLogger) Println(v ...interface{})               { log.Println(v...) }
-
-var Logger StdLogger = &defaultLogger{}
```

Comment:
Presumably, for backwards compat you'll have to keep the global Logger object, and use it as the default (fallback) if the cluster/session logger is nil? That is, inside NewCluster/NewSession you'd set the cluster's logger variable based on the config, falling back to the global.

Otherwise it would be an API-breaking change.

Reply:
ugh....right. Thanks for pointing out!

I'll look into this on Friday.